store.html

```diff
@@ -0,0 +1,118 @@
+</head>
+<body>
+    <header>
+        <a class="nav-list" href="index.html"><img src="/img/karma-logo.svg" alt="karma logo"></a>
+        <nav>
+            <ul class="nav-list">
+                <li class="nav-items" ><a class="nav-links" href="#">Meet Karma</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">How it Works</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Store</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Blog</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Help</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="form">
+            <h1 >Order your Karma wifi<br> device today!</h1>
+
+            <form action="https://api.web3forms.com/submit" method="POST">
+                <input type="hidden" name="access_key" value="51f9e0c7-d18b-49e8-85e1-1c16d0122cba">
+                <div class="fieldset">
+                    <div class="name">
+                     <label for="f-name">First name *</label>
+                     <input type="text" id="f-name" name="f-name"required />
+                    </div>
+                    <div class="name">
+                      <label for="l-name">Last name *</label>
+                      <input type="text" id="l-name"  name="l-name"required/>
+                    </div>
+                </div>
+
+                <div class="address">
+                    <div class="name">
+                      <label for="address-one">Address *</label>
+                      <input type="text"id="address-one" name="address"required/>
+                    </div>
+                </div>
+
+                <div class="address">
+                    <div class="name">
+                      <label for="address-two">Address 2</label>
+                      <input type="text"id="address-two"/>
+                    </div>
+                </div>
+
+                <div class="fieldset">
+                    <div class="name">
+                      <label for="city">City *</label>
+                      <select name="city" id="city" required>
+                        <option value="" disabled selected>Select your city...</option>
+                        <option value="Oxford">Oxford</option>
+                        <option value="london">London</option>
+                        <option value="surry">surry</option>
+                      </select>
+                    </div>
+
+                    <div class="name">
+                      <label for="postcode">Postcode *</label>
+                      <input type="text"id="postcode"required/>
+                    </div>
+                </div>
+
+                <div class="radio-box">
+                  <legend>Select a color</legend>
+                  <ul>
```

Comment:
The radio inputs do not need an unordered list tag on them, they just need a label for each input